### DISCUSSION TITLE
Removing members from organization if their AD user is not active anymore

### DISCUSSION THREAD

**Discussion by Diogomrol** [community/community#23735] [Enterprise]
<p>Hi there,</p>
<p>We currently have GitHub Enterprise and are using SAML SSO enforced to all the users (linked to our Azure AD).</p>
<p>However, there is something I still couldn’t figure out: if a member leaves the company his/her AD account gets deactivated but he/she is not removed from the organization. The user is kept there and SSO still shows as linked.</p>
<p>How can we identify GitHub users whose Azure AD identity is disabled/deactivated, or, better than that, make sure members in that situation are automatically removed?</p>
<p>Thank you!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>I think you might find this of use?</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <a href="https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/github-provisioning-tutorial" target="_blank">docs.microsoft.com</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181079147-45ede2a4-9cef-4555-9c3b-ec13bb79cb5c.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://docs.microsoft.com/en-us/azure/active-directory/saas-apps/github-provisioning-tutorial" target="_blank">Tutorial: User provisioning for GitHub - Azure AD</a></h3>

<p>Learn how to configure Azure Active Directory to automatically provision and de-provision user accounts to GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If not, and you’re still having trouble, let us know. Either here, or by contacting <a href="https://support.github.com/">Support</a> if your questions require discussion of account specific information.</p>
